BensonWollmanWeselyZombieProject/src/main/java/com/mycompany/bensonwollmanweselyzombieproject/Main.java

```diff
@@ -24,5 +26,76 @@ public static void main() {
         //Start Fight
         human1.attack_enemy(zombie1);
         zombie1.attack_enemy(human1);
+         */
+
+
+        /**
+         * Generate number of each unit and display to user
+         */
+        int numberOfHumans = (int)((Math.random() * 10) + 1);

```

Comment:
I think the decision to not include 0 (and every engineering decision) as an option should be documented in either the readme or as a comment. I don't think we need to add it now per se, but wanted to call it out.

Reply:
I think I started at one mostly for testing purposes.  This is another case where he doesn't give much direction other than "...program randomly generates an array of...".  I'm good either way though.

Reply:
Same, and agreed!